src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -888,6 +888,14 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                         if (body != null && ((methodSymbol.ContainingType.IsStructType() && !methodSymbol.IsImplicitConstructor) || instrumentForDynamicAnalysis))
                         {
+                            if (instrumentForDynamicAnalysis && methodSymbol.IsImplicitConstructor)
+                            {
+                                // Flow analysis over the initializers is necessary in order to find assignments to fields.
+                                // Bodies of implicit constructors do not get flow analysis later, so the initializers
+                                // are analyzed here.
+                                DataFlowPass.Analyze(_compilation, methodSymbol, analyzedInitializers, diagsForCurrentMethod, requireOutParamsAssigned: false);
```

Comment:
I see, thanks.

---
In reply to: [76301656](https://github.com/dotnet/roslyn/pull/13352#discussion_r76301656) [](ancestors = 76301656)